cLIMS/organization/export.py

```diff
@@ -1225,7 +1225,12 @@ def removeDup(dcicExcelSheet):
                 
 @login_required 
 def exportDCIC(request,prj_pk):
-    expPks = request.POST.getlist('dcic')
+    input_keys = [key for key in request.POST if key.startswith("dcic")]
```

Comment:
@Snehal1894 that is the fix for the DCIC issue that @betulakgol mentioned , right ?

what's the nature of the bug ?

Reply:
Yes, Initially in ExportDCIC, html page showed all experiments and if the selected exps were not in any Experiment set, it would not generate DCIC excel sheet
So we are now displaying Experiment Sets and the corresponding experiments on html page.
So the code is modified to handle experiments according to the Experiment Sets.